guide/install.md

```diff
@@ -74,7 +74,7 @@ We provide these installers to make it convenient for non-developers to join Spa
 
 ## Step 2 - Verify the Installer
 
-Follow [these steps](auth.md) to verify that the Installer you have downloaded is authentic and not has not been modified by a third party. Don't trust, verify!
+You can follow [these steps](auth.md) to verify that the Installer you have downloaded is authentic and not has not been modified by a third party.
```

Comment:
Installer or installer?
We should state that this step is optional.

Reply:
good rephrasing. The installer is an artifact / thing therefore I think that capitalizing it here is okay. Yes, but with your rewording `you can...` we communicate that this is optional.

Reply:
Note that a paragraph above and a paragraph below we don't capitalize this word.
Since we aim that anyone would be able to follow and complete the installation, I think that it would be good to explicitly state that it's optional. If you disagree, at lease last change "can" to "may".

Reply:
Please go and make the changes you see fit here.

Reply:
Updated.